src/components/Farmhand/helpers/getPlayerInventoryQuantities.js

```diff
@@ -0,0 +1,16 @@
+import { itemsMap } from '../../../data/maps'
+
+const itemIds = Object.keys(itemsMap)
+
+/**
+ * @param {Array.<{ id: farmhand.item, quantity: number }>} inventory
+ * @returns {Object.<string, number>}
+ */
+export const getPlayerInventoryQuantities = inventory => {
```

Comment:
does this need the word "Player" in it? seems like any inventory could come through and work with this method (though maybe today the player's inventory is the only one that exists)